e2e/npm_link_package-esm/MODULE.bazel

```diff
@@ -0,0 +1,6 @@
+###############################################################################
```

Comment:
Delete...

Reply:
✅

Reply:
It's still there...

Reply:
🤦

Reply:
huh... git commit says they are removed
```$ gd HEAD~ --name-only
.bazelrc
WORKSPACE
WORKSPACE.bzlmod
js/dev_repositories.bzl
```
I think the GH UI is being buggy here

Reply:
There we go. Pushed a few more times and it updated correctly.